repo/fsrepo.go

```diff
@@ -67,41 +69,42 @@ type FSRepo struct {
 
 var _ Repo = (*FSRepo)(nil)
 
-// GetRepoDir is a helper for either using a user provided repoDir or fetching
-// the repoDir from FSRepoDir
-func GetRepoDir(repoDir string) string {
-	if repoDir == "" {
-		repoDir = FSRepoDir()
+// GetRootDir is a helper for either using a user provided repo root directory
+// or fetching the root directory from FSRepoDir.
+func GetHomeDir(homeDir string) string {
+	if homeDir == "" {
+		homeDir = FSHomeDir()
 	}
-	return repoDir
+	return homeDir
 }
 
 // FSRepoDir is a helper for getting the path to the repodir
-func FSRepoDir() string {
-	envRepoDir := os.Getenv("FIL_PATH")
-	if envRepoDir != "" {
-		return envRepoDir
+func FSHomeDir() string {
+	envHomeDir := os.Getenv("FIL_PATH")
+	if envHomeDir != "" {
+		return envHomeDir
 	}
 	return "~/.filecoin"
 }
 
 // CreateRepo provides a quick shorthand for initializing and opening a repo
-func CreateRepo(repoDir string, cfg *config.Config) (*FSRepo, error) {
-	repoDir = GetRepoDir(repoDir)
-	if err := InitFSRepo(repoDir, cfg); err != nil {
+func CreateRepo(rootDir string, cfg *config.Config) (*FSRepo, error) {
+	rootDir = GetHomeDir(rootDir)
+	if err := InitFSRepo(rootDir, cfg); err != nil {
 		return nil, err
 	}
-	return OpenFSRepo(repoDir)
+	return OpenFSRepo(rootDir)
 }
 
 // OpenFSRepo opens an already initialized fsrepo at the given path
 func OpenFSRepo(p string) (*FSRepo, error) {
```

Comment:
Change `p` and the comment to clearly refer to `homePath` not the repo path.

Possibly better would be to take a repo path here and push the expansion and joining up to the caller.